### ISSUE TITLE
Anonymous types(dynamic/expando objects) and JSON

### ISSUE THREAD

**Issue by Sl4vP0weR** [Yellow-Dog-Man/Resonite-Issues#2696]
### Is your feature request related to a problem? Please describe.

When we use network nodes, we mostly work with JSON content, either sending it or receiving it.
It's hard to parse JSON content using nodes that are currently accessible; the node count required for parsing even a small web response is enormous.
JSON objects could also be used to store multiple values within one object, which is helpful when you try to save/load data.

### Describe the solution you'd like

.NET supports dynamic/expando objects, which are generally dictionaries.
I imagine using a node `ParseJSONObject` and getting/setting values from it using a node `GetJSONProperty`/`SetJSONProperty`. 
This can be implemented easily using Newtonsoft.Json or System.Text.Json by evaluating a parsed JSON object to an dynamic/expando object.

### Describe alternatives you've considered

Parsing everything using `IndexOf` and `Substring` is clearly not the best way to do it.

### Requesters

Sl4vP0weR

**Comment by XDelta** [comment]
Mostly duplicate requests for JSON parsing https://github.com/Yellow-Dog-Man/Resonite-Issues/issues/946 and Collections https://github.com/Yellow-Dog-Man/Resonite-Issues/issues/572
